src/mixins/stylable.js

```diff
@@ -0,0 +1,12 @@
+import StylePropable from './style-propable';
+
+export default function(target) {
+  target.propTypes = {
+    ...StylePropable.propTypes,
+    ...target.propTypes,
+  };
+
+  target.prototype.mergeStyles = StylePropable.mergeStyles;
```

Comment:
Side not: this is an utility function. I think that we should move it out of the decorator at some point.